src/request/components/request-detail-panel-logging.jsx

```diff
@@ -55,22 +80,22 @@ var LogMessages = React.createClass({
                 <thead>
                     <tr className="table-col-title-group">
                         <th width="5%"><span className="table-col-title">#</span></th>
-                        <th width="10%"><span className="table-col-title">Level</span></th>
+                        <th width="15%"><span className="table-col-title"><Icon fixedWidth="true" /> Level</span></th>
                         <th><span className="table-col-title">Message</span></th>
-                        <th width="10%"><span className="table-col-title">From Start</span></th>
-                        <th width="10%"><span className="table-col-title">Duration</span></th>
+                        <th width="15%"><span className="table-col-title">From Start (ms)</span></th>
+                        <th width="15%"><span className="table-col-title">Duration (ms)</span></th>
```

Comment:
IMO, a diagnostics tools should *always* indicate its units to disambiguate comparisons between values.  I don't have a strong preference for whether that's at a column level (which tends to reduce clutter) or at a value level (which tends to improve readability and understanding).  (The same goes for timestamps, which should always indicate whether a time is local or UTC.)